docs/related-tools/local-analysis/client-side-tools.md

```diff
@@ -0,0 +1,30 @@
+# Client-side tools
+
+Client-side tools enable you to run analysis either locally or as part of your CI process and integrating the results into your Codacy workflow. This way, Codacy presents the analysis information reported by your local tools alongside all other code quality information on the Codacy dashboards.
+
+Codacy supports client-side tools in two ways:
+
+-   **Standalone tools**
+
+    Codacy provides auxiliary converters that parse the output of third-party tools, convert the output format, and upload the results to Codacy. You must download, configure, and run the third-party tools yourself.
+
+    You can't configure these tools on the Codacy UI, since you manage their configuration locally.
+
+-   **Containerized tools**
```

Comment:
Which tools are containerized and which are not?

Reply:
Currently, I list all tools with an indication if they are "standalone" or "containerized" at the bottom of the page.

I did experiment with having separate lists for the two types of tools. However, I figured that customers will not choose the tools they will run based on their type, but depending on the programming languages they're using and their needs. Because of this, I decided to provide a simple list with all the tools (pointing to self-contained instructions on how to run them) to make the choice and setup of the tools as transparent as possible.